inspire_disambiguation/api.py

```diff
@@ -198,34 +218,43 @@ def cluster(
             ),
             signature_block=phonetic_block,
             B3_f_score=cluster.supervised_scoring(clusterer.y, cluster.labels_)
-            if hasattr(cluster, "supervised_scoring")
+            if hasattr(cluster, "supervised_scoring") and not test_labels
```

Comment:
I excluded only this statistic because others (training time, threshold) might be useful and I don't want to duplicate the code in the ```LOGGER.info()``` below